engine/src/cmd/computer.cpp

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
The [GPL](https://github.com/vegastrike/Vega-Strike-Engine-Source/blob/master/LICENSE) says,

----

  If you develop a new program, and you want it to be of the greatest
possible use to the public, the best way to achieve this is to make it
free software which everyone can redistribute and change under these terms.

  To do so, attach the following notices to the program.  It is safest
to attach them to the start of each source file to most effectively
state the exclusion of warranty; and each file should have at least
the "copyright" line and a pointer to where the full notice is found.

    <one line to give the program's name and a brief idea of what it does.>
    Copyright (C) <year>  <name of author>

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.

Also add information on how to contact you by electronic and paper mail.

----

Roy, if you have time, can you add the above notice at the top of each new source file you added? Hopefully it will just be a copy-paste job.

Reply:
@BenjamenMeyer can we do this as a separate issue. I just checked and planet_generic.cpp and unit_generic.cpp don't have it either. So we should probably do all at the same time.

Reply:
> @BenjamenMeyer can we do this as a separate issue. I just checked and planet_generic.cpp and unit_generic.cpp don't have it either. So we should probably do all at the same time.

That seems reasonable.